OpenRA.Mods.Common/LoadScreens/BlankLoadScreen.cs

```diff
@@ -64,7 +64,7 @@ public void StartGame(Arguments args)
 			var connect = args != null ? args.GetValue("Launch.Connect", null) : null;
 			if (!string.IsNullOrEmpty(connect))
 			{
-				var parts = connect.Split(':');
+				var parts = connect.Replace("openra://", "").Split(':');
```

Comment:
`launch-replay.sh` is neither installed nor used for any of this. I can't do any filtering there.